src/components/likedByContainer/LikedByContainer.tsx

```diff
@@ -1,40 +1,64 @@
-import {
-    SAMPLE1_LIKED_BY_PROFILE_PICTURE,
-    SAMPLE2_LIKED_BY_PROFILE_PICTURE,
-} from '../../constants';
+import { MOCK_DATA } from '../../mockText';
+import { AND, LIKED_BY, OTHER, OTHERS, PROFILE_PICTURE_ALT } from '../../constants';
+import { st, classes } from './LikedByContainer.st.css';
+import { classes as generalClasses } from '../../style/general.st.css';
 import type { User } from '../../types';
 
 interface ILikedByContainer {
     allUsersWhoLiked: User[];
 }
 
 const LikedByContainer = ({ allUsersWhoLiked }: ILikedByContainer) => (
-    <div>
-        <div>
-            <div>
-                <img src={SAMPLE1_LIKED_BY_PROFILE_PICTURE} alt="profile Picture"></img>
+    <div className={classes.root}>
+        <div className={classes.likeByWho}>
+            <div className={classes.UserLikeByPic}>
+                {allUsersWhoLiked.map((user, index) => {
```

Comment:
why not just?

const usersToDisplay = allUsersWhoLiked.slice(1, 2)

Reply:
show only 2 dont go over all array